spoon-runner/src/main/java/com/squareup/spoon/SpoonUtils.java

```diff
@@ -9,6 +9,9 @@
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonWriter;
 import com.madgag.gif.fmsware.AnimatedGifEncoder;
+import org.apache.commons.io.FileUtils;
+
+import javax.imageio.ImageIO;
```

Comment:
Please revert.  We keep regular imports in a simple alphabetical list, with no newlines between them.